_docs/style/interactive-states.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Interactive States
+layout: docs
+category: Style
+usage: >
+  The system of interactive states ensures consistent styles for the states of controls in Helix. This system accounts for multiple types of controls, with various interactive patterns, and defines commonalities and standards across all of them.
+
+  All controls that live in the application body strictly adhere to subsets of the specifications in this document, unless otherwise noted in their individual design docs.
```

Comment:
Maybe move this to the front of the sentence "Unless otherwise noted in their individual design docs, all controls...."

Reply:
done.